### ISSUE TITLE
[Accessibility, a11y] Quick picks select many - unable to reach the `Toggle All Checkboxes` control

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#263645]
**VS Code Version**: 1.104.0-insider

**OS Version**: macOS Sequoia 15.6.1 
(Source issue also states that it reproduces on Windows)

**Screenreader**: VoiceOver

**Extensions Installed**:
Put together an extension to quickly demo the bare VS Code UI features without our normal SDK and extension wrappers.

"a11y-kit" (https://marketplace.visualstudio.com/items?itemName=MicroFish.accessibility-testkit)

**Repro Steps**:

1. Install `a11y-kit` from VS Code Marketplace
2. F1 > `a11y-kit: Select UI feature for accessibility testing` > `Show Quick Picks - Many`
3. Try to select the toggle for all checkboxes

**Expected**
"Toggle all checkboxes" can be navigated to using only keyboard.

**Actual**
"Toggle all checkboxes" control not accessible with only keyboard.

<img width="884" height="208" alt="Image" src="https://github.com/user-attachments/assets/d4dc53aa-0d78-482b-80c3-7eaf58f2b970" />

**Original Source**:
https://github.com/microsoft/vscode-azurefunctions/issues/4678

**Comment by meganrogge** [comment]
This was fixed recently, see how I moved my focus there with `Shift+tab`. pls try the latest

<img width="938" height="542" alt="Image" src="https://github.com/user-attachments/assets/36c2aefd-b425-4294-8c18-a33670ba9588" />
